COMPILING.md

```diff
@@ -1,3 +1,11 @@
+# BEFORE COMPILATION
+
+JBMC uses a git submodule. This submodule must be set up before
+attempting a build:
+```
+git submodule update --init --recursive
```

Comment:
The `--recursive` part is strictly speaking not necessary as of now. It doesn't _hurt_ to have in there though, so no real objection.